services-camel/src/main/java/edu/unc/lib/dl/services/camel/util/CdrFcrepoHeaders.java

```diff
@@ -27,6 +27,8 @@
 
     public static final String CdrBinaryPath = "CdrBinaryPath";
 
+    public static final String CdrBinaryId = "CdrBinaryId";
```

Comment:
Don't think this is being used anywhere anymore, other than being initialized in tests from when it was previously used. So I think you can go ahead and remove it